concourse/pipelines/certification_pipeline.yml

```
@@ -8,12 +8,7 @@ anchors:
     put: slack-alert
     params:
       text: |
-        <((ud/pxf/secrets/ud-concourse-url))/builds/$BUILD_ID|$BUILD_PIPELINE_NAME/$BUILD_JOB_NAME> went red :blob_slightly_frowning_face:
-  on_success:
-    put: slack-alert
-    params:
-      text: |
-        <((ud/pxf/secrets/ud-concourse-url))/builds/$BUILD_ID|$BUILD_PIPELINE_NAME/$BUILD_JOB_NAME> went green! :smile:
+        :ci-fail: <((ud/pxf/secrets/ud-concourse-url))/builds/$BUILD_ID|$BUILD_PIPELINE_NAME/$BUILD_JOB_NAME> has failed (<((ud/pxf/common/slack-em-id.id1))>, <((ud/pxf/common/slack-em-id.id1))>)
```

shouldn't this be `id1` and `id2`?
```suggestion
        :ci-fail: <((ud/pxf/secrets/ud-concourse-url))/builds/$BUILD_ID|$BUILD_PIPELINE_NAME/$BUILD_JOB_NAME> has failed (<((ud/pxf/common/slack-em-id.id1))>, <((ud/pxf/common/slack-em-id.id2))>)